README.md

```diff
@@ -1,27 +1,148 @@
+- Support for mapping groupings of columns to a single visual aesthetic
 - A high-level [_Grammar of Graphics_](http://vita.had.co.nz/papers/layered-grammar.pdf) style API that can specify a wide variety of visualizations with a few simple concepts
 - A [columnar](https://observablehq.com/@mbostock/manipulating-flat-arrays) interface for input data that enables efficient interop with Web Workers and [Apache Arrow](https://arrow.apache.org/)
-- Easy reactivity and extensibility via React
 - Self-contained configs in the style of [Vega-Lite](https://vega.github.io/vega-lite/)
-- Support for mapping groupings of columns to a single visual aesthetic
 
-## Development
+## Getting Started [](#getting-started)
+
+#### Installation
+
+Install [Giraffe](https://www.npmjs.com/package/@influxdata/giraffe) with your package manager
+
+`yarn add @influxdata/giraffe` or `npm install @influxdata/giraffe`
+
+#### Example
+
+1. In your React code, import the `Plot` component and the `newTable` utility function
+
+  <pre>
+  import {Plot, newTable} from '@influxdata/giraffe'
+  </pre>
+
+2. Build the config object.  
+   a. **Required properties**:
+
+   - `table` is data built using the newTable utilty function (_also_ built from Flux results, _see_ [Flux example](./README.md#example-using-flux))
+   - `layers` is an array of objects that describe how to render the data.
+
+   b. _Optional properties_ include customizations for
+
+   - gridlines: color and opacity
+   - axes: appearance, color, opacity, and scaling
+   - ticks: generation, formatting and labeling, font, and color
+   - legend (tooltip): labeling and styling
+
+   For details on all configuration properties, go to the [configuration guide](./giraffe/README.md#config).
+
+   Here is an example of building the config object while skipping optional properties:
+
+  <pre>
+  // Example table and layer
+
+  const table = newTable(5)
+    .addColumn('_time', 'time', [1589838401244, 1589838461244, 1589838521244, 1589838581244, 1589838641244])
+    .addColumn('_value', 'number', [2.58, 7.11, 4.79, 8.89, 2.23])
 
-See [`CONTRIBUTING.md`](./CONTRIBUTING.md).
+  const lineLayer = {
+    type: "line",
+    x: "_time",
+    y: "_value",
+  }
 
-## Thanks
+  const config = {
+    table: table,
+    layers: [lineLayer],
+  }
+  </pre>
+
+3. Render your component by passing the `confg` object as the config prop to the `<Plot>` component. Be sure that the parent component around `<Plot>` has both a height and a width measured in positive values. If either is not a positive value, the graph will not be visible.
+
+   For example, to make a `<Plot>` that adjusts to screen height and width, in your React rendering code return this element:
+
+  <pre>
+  // return this element in your React rendering code:
+
+  &#60;div
+    style={{
+      width: "calc(70vw - 20px)",
+      height: "calc(70vh - 20px)",
+      margin: "40px",
+    }}
+  &#62;
+    &#60;Plot config={config} /&#62;
+  &#60;/div&#62;
```

Comment:
We can't use actual divs inside a `<pre>` if we want to show the literal "`<div>`" as text. I chose `<pre>` over triple \` marks because I wanted to indent the whole code block, which `<pre>` allows me to do while \` marks do not